src/main/java/mate/academy/BookAppApplication.java

```diff
@@ -1,13 +1,37 @@
 package mate.academy;
 
+import java.math.BigDecimal;
+import mate.academy.model.Book;
+import mate.academy.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
 @SpringBootApplication
 public class BookAppApplication {
+    private BookService bookService;
+
+    @Autowired
```

Comment:
If we need to inject just one field we can omit this annotation, but it's not a problem

Reply:
fixed =)